app/src/main/java/nl/tudelft/bbw/controller/CryptoController.java

```diff
@@ -58,4 +64,37 @@ public final String decryptString(String data) {
             throw new RuntimeException("Error - Could not decode string: " + e);
         }
     }
+
+    /**
+     * encryptFile method
```

Comment:
It is not per definition bad and thus consistency is very important as well. Next to that, if you have a look at : https://github.com/deepweeb/B-B-W/tree/master/app/src/main/java/nl/tudelft/b_b_w, almost all controllers have them and almost all models too.